roles/ansible-keylime-tpm20/vars/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+# section for vars
+shell_profiles:
+- .bash_profile
+
+# version variables
+tpm2_tss_version: a0d6f7d
```

Comment:
could you add this for tpm2-tools and tpm2-abrmd as well please?

    tpm2_tools_version: master
    tpm2_abrmd_version: master

And of course, add the two above to `git-repos.yml`

I guess eventually we could move to using dnf / yum to install these, when things stabilise more, but for now its useful to test against branches.

Reply:
Sure! I will make this change :)